sdk/avalon_sdk/direct/avalon_direct_client.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, config_file=None, config=None):
                     raise Exception('Could not open config file: %s' % e)
 
         self.__blockchain_type = self.__config['blockchain']['type']
-        if self.__blockchain_type == "Ethereum":
+        if self.__blockchain_type == "ethereum":
```

Comment:
May be do this check -> self.__blockchain_type.lower() == 'ethereum'

Reply:
Done